.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+**[Mailing List](https://groups.google.com/forum/#!forum/liftweb) thread:** REPLACE THIS WITH ML LINK
+
+**Example project link:** LINK TO EXAMPLE PROJECT IF APPLICABLE
```

Comment:
We certainly can yeah, perhaps under `docs/community` or something? Or move it to the Lift site…